restord/ImageSessionServer.m

```diff
@@ -350,8 +351,13 @@ - (void)processOutput:(NSData *)output {
 // Mount and unmounts may take up to 5 minutes.
 //
 
-// Used by both mountDisk and unmountDisk to wait for mount/unmount to complete before returning.
-void MountUnmountCallback(DADiskRef disk, DADissenterRef dissenter, void *context) {
+// Used by various disk operations to wait for completion before returning.
+void MountUnmountEjectCallback(DADiskRef disk, DADissenterRef dissenter, void *context) {
+  if (dissenter) {
+    NSLog(@"MountUnmountEjectCallback: Error from Unmount %s: status=%X, string=%@", DADiskGetBSDName(disk),
```

Comment:
nit: When the format string is long, it's a bit more readable to put the args on new lines.

```
    NSLog(@"MountUnmountEjectCallback: Error from Unmount %s: status=%X, string=%@",
          DADiskGetBSDName(disk), DADissenterGetStatus(dissenter),
          DADissenterGetStatusString(dissenter));
```

Reply:
Done